tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/js/operator-finder/operator-finder.js

```diff
@@ -126,7 +127,80 @@ define(['jquery', 'lodash', 'log', 'handlebar', 'designViewUtils', 'app/source-e
         };
 
         /**
-         * Flattens the metadata structure into an array ti reduce the search complexity.
+         * replacing the pipeline and newline character for md conversation.
+         * @param data string variable
+         * @returns {""} retrun string data
+         */
+        var pipelineNewlineEscape = function (data) {
+            return data.replace(/[|]/g, '&#124;').replace(/[\n]/g, '<br/>');
+        }
+
+        /**
+         * mdconvertion which converst md data into html.
+         * @param cloneOperator extension object
+         * @returns {*{}} return extension object which contains converted md
+         */
+        var mdConversion = function (cloneOperator) {
+            //Add remarkable instance to convert md type data into html
+            var markDownConvertor = new Remarkable({
+                html: true, // Enable HTML tags in source
+                xhtmlOut: false, // Use '/' to close single tags (<br />)
+                breaks: false, // Convert '\n' in paragraphs into <br>
+                linkify: true, // Autoconvert URL-like text to links
+                // Enable some language-neutral replacement + quotes beautification
+                typographer: false,
+                // Double + single quotes replacement pairs, when typographer enabled,
+                // and smartquotes on. Set doubles to '«»' for Russian, '„“' for German.
+                quotes: '“”‘’',
+                highlight: function (/*str, lang*/) {
+                    return '';
+                }
+            });
+            if (cloneOperator.description) {
+                cloneOperator.extensionDescription = markDownConvertor.render(cloneOperator.description);
+            }
+            if (cloneOperator.examples) {
+                cloneOperator.example = "";
```

Comment:
```suggestion
                cloneOperator.combinedExamples = "";
```

Reply:
960249df47be3bbdc76d0016d7e1dd9ac324ac05 this commit fix above request change